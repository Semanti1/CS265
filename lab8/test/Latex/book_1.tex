%
\documentclass[11pt]{book}
%\pagestyle{plain}
\usepackage{makeidx}
%\usepackage{graphicx}

\usepackage{multirow} % Needed for cells in tables that span rows
\usepackage{listings} % for \lstset , listing code (chptr-code-listing)
\usepackage{color}   % make code pretty
\usepackage{multicol} % for a multi-column list
\usepackage{hyperref} % for \url , \href , etc.
\usepackage{framed} % for \framed
%\usepackage{pdfpages} % for \includepdf

%?? - Huh?  Can't get \k{} sequence
%\usepackage[T1]{fontenc}
%\usepackage[utf8]{inputenc}

%------- lstsetlisting settings  -------------

\definecolor{dkgreen}{rgb}{0,0.6,0}
\definecolor{gray}{rgb}{0.5,0.5,0.5}
\definecolor{dkred}{rgb}{0.8, 0, 0}
\definecolor{seablue}{rgb}{0, 0.5, 1}

\lstset{%
  basicstyle={\small\ttfamily},
	frame=single,
  language=C,
  aboveskip=3mm,
  belowskip=3mm,
  showstringspaces=false,
  columns=flexible,
  numbers=none,
  numberstyle=\tiny\color{blue},
	%identifierstyle=\color{seablue},
  keywordstyle=\color{dkgreen},
  commentstyle=\color{gray},
  stringstyle=\color{dkred},
  breaklines=false,
	resetmargins=true,
  %breakatwhitespace=true,
  tabsize=3
}


%%%%%%  Top Matter  %%%%%%%%%%%%%%%
\title{First Book}
\author{Kurt Schmidt}
\date{Aug 2014}

\begin{document}

\maketitle % ?? Use this for books?

%%%%%%  Front Matter  %%%%%%%%%%%%%%%
\frontmatter
\tableofcontents

%%%%%%  Main Matter  %%%%%%%%%%%%%%%
\mainmatter
\include{chptr01}
\include{chptr-basics}
\include{chptr-math}
\include{chptr-tables}
\include{chptr-code-listing}

%%%%%%  Back Matter  %%%%%%%%%%%%%%%
\backmatter
%TODO get index working
\printindex  % Automagic?  Ah, needed the package.  Still doing nothing.
%bibliography.  TODO.  Not there yet

\end{document}
