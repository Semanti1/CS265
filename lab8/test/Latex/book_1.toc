\contentsline {chapter}{\numberline {1}In a Galaxy, Far Far Away}{1}{chapter.1}
\contentsline {section}{\numberline {1.1}Notes}{1}{section.1.1}
\contentsline {section}{\numberline {1.2}Heather Graham}{1}{section.1.2}
\contentsline {section}{\numberline {1.3}Compiling}{1}{section.1.3}
\contentsline {subsection}{\numberline {1.3.1}Compiler Warnings and Errors}{2}{subsection.1.3.1}
\contentsline {chapter}{\numberline {2}Basics}{3}{chapter.2}
\contentsline {section}{\numberline {2.1}Hello, World}{3}{section.2.1}
\contentsline {section}{\numberline {2.2}Modes}{4}{section.2.2}
\contentsline {section}{\numberline {2.3}Metacharacters}{5}{section.2.3}
\contentsline {subsection}{\numberline {2.3.1}Braces for Grouping}{5}{subsection.2.3.1}
\contentsline {subsection}{\numberline {2.3.2}Spaces}{5}{subsection.2.3.2}
\contentsline {subsection}{\numberline {2.3.3}Quotes}{6}{subsection.2.3.3}
\contentsline {subsection}{\numberline {2.3.4}Dashes and Hyphens}{6}{subsection.2.3.4}
\contentsline {section}{\numberline {2.4}Special Characters}{6}{section.2.4}
\contentsline {subsection}{\numberline {2.4.1}Composition and Decoration}{6}{subsection.2.4.1}
\contentsline {subsection}{\numberline {2.4.2}Other Defined Characters and Symbols}{7}{subsection.2.4.2}
\contentsline {subsection}{\numberline {2.4.3}Common Symbols and Characters from Packages}{7}{subsection.2.4.3}
\contentsline {subsection}{\numberline {2.4.4}Math Symbols}{8}{subsection.2.4.4}
\contentsline {chapter}{\numberline {3}Math}{11}{chapter.3}
\contentsline {chapter}{\numberline {4}Tables}{13}{chapter.4}
\contentsline {section}{\numberline {4.1}Simple Tables}{13}{section.4.1}
\contentsline {subsection}{\numberline {4.1.1}Vertical Lines}{13}{subsection.4.1.1}
\contentsline {subsection}{\numberline {4.1.2}Content of Tables}{13}{subsection.4.1.2}
\contentsline {section}{\numberline {4.2}Wrapping Text}{14}{section.4.2}
\contentsline {section}{\numberline {4.3}Aligning on the Radix}{14}{section.4.3}
\contentsline {subsection}{\numberline {4.3.1}@-Expressions}{14}{subsection.4.3.1}
\contentsline {subsection}{\numberline {4.3.2}Tables of Numbers}{14}{subsection.4.3.2}
\contentsline {section}{\numberline {4.4}Spanning}{15}{section.4.4}
\contentsline {subsection}{\numberline {4.4.1}Cell Spanning Multiple Columns}{15}{subsection.4.4.1}
\contentsline {subsection}{\numberline {4.4.2}Cell Spanning Multiple Rows}{15}{subsection.4.4.2}
\contentsline {section}{\numberline {4.5}Don't know yet}{16}{section.4.5}
\contentsline {chapter}{\numberline {5}Embedding Code in Text}{17}{chapter.5}
\contentsline {section}{\numberline {5.1}Inlining Code in Text}{17}{section.5.1}
\contentsline {subsection}{\numberline {5.1.1} Define Your Own Command }{17}{subsection.5.1.1}
\contentsline {section}{\numberline {5.2} Blocks of code }{17}{section.5.2}
\contentsline {subsection}{\numberline {5.2.1} \texttt {verbatim} Environment }{17}{subsection.5.2.1}
\contentsline {subsection}{\numberline {5.2.2} The \texttt {listings} Package }{18}{subsection.5.2.2}
\contentsline {subsection}{\numberline {5.2.3}Language Supported by \texttt {listings}}{20}{subsection.5.2.3}
