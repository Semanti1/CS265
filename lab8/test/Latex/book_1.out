\BOOKMARK [0][-]{chapter.1}{In a Galaxy, Far Far Away}{}
\BOOKMARK [1][-]{section.1.1}{Notes}{chapter.1}
\BOOKMARK [1][-]{section.1.2}{Heather Graham}{chapter.1}
\BOOKMARK [1][-]{section.1.3}{Compiling}{chapter.1}
\BOOKMARK [2][-]{subsection.1.3.1}{Compiler Warnings and Errors}{section.1.3}
\BOOKMARK [0][-]{chapter.2}{Basics}{}
\BOOKMARK [1][-]{section.2.1}{Hello, World}{chapter.2}
\BOOKMARK [1][-]{section.2.2}{Modes}{chapter.2}
\BOOKMARK [1][-]{section.2.3}{Metacharacters}{chapter.2}
\BOOKMARK [2][-]{subsection.2.3.1}{Braces for Grouping}{section.2.3}
\BOOKMARK [2][-]{subsection.2.3.2}{Spaces}{section.2.3}
\BOOKMARK [2][-]{subsection.2.3.3}{Quotes}{section.2.3}
\BOOKMARK [2][-]{subsection.2.3.4}{Dashes and Hyphens}{section.2.3}
\BOOKMARK [1][-]{section.2.4}{Special Characters}{chapter.2}
\BOOKMARK [2][-]{subsection.2.4.1}{Composition and Decoration}{section.2.4}
\BOOKMARK [2][-]{subsection.2.4.2}{Other Defined Characters and Symbols}{section.2.4}
\BOOKMARK [2][-]{subsection.2.4.3}{Common Symbols and Characters from Packages}{section.2.4}
\BOOKMARK [2][-]{subsection.2.4.4}{Math Symbols}{section.2.4}
\BOOKMARK [0][-]{chapter.3}{Math}{}
\BOOKMARK [0][-]{chapter.4}{Tables}{}
\BOOKMARK [1][-]{section.4.1}{Simple Tables}{chapter.4}
\BOOKMARK [2][-]{subsection.4.1.1}{Vertical Lines}{section.4.1}
\BOOKMARK [2][-]{subsection.4.1.2}{Content of Tables}{section.4.1}
\BOOKMARK [1][-]{section.4.2}{Wrapping Text}{chapter.4}
\BOOKMARK [1][-]{section.4.3}{Aligning on the Radix}{chapter.4}
\BOOKMARK [2][-]{subsection.4.3.1}{@-Expressions}{section.4.3}
\BOOKMARK [2][-]{subsection.4.3.2}{Tables of Numbers}{section.4.3}
\BOOKMARK [1][-]{section.4.4}{Spanning}{chapter.4}
\BOOKMARK [2][-]{subsection.4.4.1}{Cell Spanning Multiple Columns}{section.4.4}
\BOOKMARK [2][-]{subsection.4.4.2}{Cell Spanning Multiple Rows}{section.4.4}
\BOOKMARK [1][-]{section.4.5}{Don't know yet}{chapter.4}
\BOOKMARK [0][-]{chapter.5}{Embedding Code in Text}{}
\BOOKMARK [1][-]{section.5.1}{Inlining Code in Text}{chapter.5}
\BOOKMARK [2][-]{subsection.5.1.1}{ Define Your Own Command }{section.5.1}
\BOOKMARK [1][-]{section.5.2}{ Blocks of code }{chapter.5}
\BOOKMARK [2][-]{subsection.5.2.1}{ verbatim Environment }{section.5.2}
\BOOKMARK [2][-]{subsection.5.2.2}{ The listings Package }{section.5.2}
\BOOKMARK [2][-]{subsection.5.2.3}{Language Supported by listings}{section.5.2}
